Player/desktopplayer.js

```diff
@@ -32,11 +32,12 @@ function SaveTranscript(data) {
     transcriptString += data;
 }
 
-// Added by KV to write/append to GAMENAME-log.txt in Documents\Quest Logs
+// Write/append to GAMENAME-log.txt in Documents\Quest Logs
 function WriteToLog(data) {
-    if (!webPlayer && data != '' && typeof (data) == 'string') {
-        UIEvent("WriteToLog", getTimeAndDateForLog() + " " + data);
+    if (typeof (data) != 'string') {
+        data = '';

```

Comment:
Do we really want to write an empty log in this case? Would it better to just return from the function and not log anything, or log some error message?

Reply:
> Would it better to just return from the function and not log anything, or log some error message?

That's how it is before the changes, but I noticed the old log window prints those as blank entries. In fact, the blank entry was missing from the log file while I was testing something earlier, and I would have missed an error had I not seen the blank entry in the actual log window.

I'm fine with whatever you think concerning this, though.

Reply:
Maybe set data to `typeof(data)`, then it's hopefully obvious to the game author that they messed up.